external/deep-object-reid/torchreid_tasks/utils.py

```diff
@@ -54,6 +54,7 @@
 
 from torchreid.utils import set_model_attr, get_model_attr
 
+
```

Comment:
Should we use this class even if we use MPATemplate?

Reply:
No, we don't need this. I'll remove these unnecessary things in future PR. (Remove torchreid dependency)